oberon/drivers/oberon_ecdsa.c

```diff
@@ -477,7 +477,7 @@ psa_status_t oberon_ecdsa_verify_message(
     psa_key_type_t type = psa_get_key_type(attributes);
 
     switch (type) {
-#if defined(PSA_NEED_OBERON_PURE_EDDSA_TWISTED_EDWARDS_255) || defined(PSA_NEED_OBERON_PURE_EDDSA_TWISTED_EDWARDS_448)
+#if (defined(PSA_NEED_OBERON_PURE_EDDSA_TWISTED_EDWARDS_255) || defined(PSA_NEED_OBERON_PURE_EDDSA_TWISTED_EDWARDS_448)) && !defined(PSA_NEED_OBERON_ED25519PH)
     case PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY_TWISTED_EDWARDS):
     case PSA_KEY_TYPE_ECC_PUBLIC_KEY(PSA_ECC_FAMILY_TWISTED_EDWARDS):
         // EDDSA is only available in verify_message
```

Comment:
If you add a check here to make sure the algorithm is EDDSA PURE it should work. Something like:
`if (alg != PSA_ALG_PURE_EDDSA) return PSA_ERROR_NOT_SUPPORTED;`

Reply:
Do you mean `if (alg != PSA_ALG_ED25519PH)`? We do want to support PURE EDDSA still no?